Need to run `npm i` in root folder before trying example

<!--
Thank you for reporting an issue.

This issue tracker is for bugs and issues found within the JavaScript implementation of libp2p.
If you require more general support please file an issue on our discuss forum. https://discuss.ipfs.io/

Please fill in as much of the template below as you're able.

Version: package.json version or the commit you have installed.
Platform: output of `uname -a` (UNIX), or version and 32 or 64-bit (Windows). If using in a Browser, please share the browser version as well.
Subsystem: if known, please specify affected core module name (e.g Transports, SECIO, etc).

If possible, please provide code that demonstrates the problem, keeping it as
simple and free of external dependencies as you are able.
-->

- **Version**: 0.31.0-rc.3
- **Platform**: Linux
- **Subsystem**: 5.10.19-1-MANJARO #1 SMP PREEMPT Fri Feb 26 11:38:34 UTC 2021 x86_64 GNU/Linux 

<!-- Bug, Feature, Question, Enhancement, Etc -->
#### Type: Bug

<!-- 
One of following:
  Critical - System crash, application panic.
  High - The main functionality of the application does not work, API breakage, repo format breakage, etc.
  Medium - A non-essential functionality does not work, performance issues, etc.
  Low - An optional functionality does not work.
  Very Low - Translation or documentation mistake. Something that won't give anyone a bad day.
-->
#### Severity: Very Low

#### Description:

The `js-libp2p/examples/libp2p-in-the-browser` readme says to run `nom install` in the example folder.
This leads to the error below.
Instead, it should say to run `npm install` in the root folder first.

I am happy to open a PR later this week.

#### Steps to reproduce the error:

<!--
This is for you! Please read, and then delete this text before posting it.
The js-libp2p issues are only for bug reports and directly actionable features.

Read https://github.com/ipfs/community/blob/master/CONTRIBUTING.md#reporting-issues if your issue doesn't fit either of those categories.
-->
```
cd js-libp2p/examples/libp2p-in-the-browser
▶ npm install
npm WARN deprecated request-promise-native@1.0.9: request-promise-native has been deprecated because it extends the now deprecated request package, see https://github.com/request/request/issues/3142
npm WARN deprecated har-validator@5.1.5: this library is no longer supported
npm WARN deprecated urix@0.1.0: Please see https://github.com/lydell/urix#deprecated
npm WARN deprecated resolve-url@0.2.1: https://github.com/lydell/resolve-url#deprecated
npm WARN deprecated debug@4.1.1: Debug versions >=3.2.0 <3.2.7 || >=4 <4.3.1 have a low-severity ReDos regression when used in a Node.js environment. It is recommended you upgrade to 3.2.7 or 4.3.1. (https://github.com/visionmedia/debug/issues/797)
npm WARN deprecated debug@4.1.1: Debug versions >=3.2.0 <3.2.7 || >=4 <4.3.1 have a low-severity ReDos regression when used in a Node.js environment. It is recommended you upgrade to 3.2.7 or 4.3.1. (https://github.com/visionmedia/debug/issues/797)
npm WARN deprecated debug@4.1.1: Debug versions >=3.2.0 <3.2.7 || >=4 <4.3.1 have a low-severity ReDos regression when used in a Node.js environment. It is recommended you upgrade to 3.2.7 or 4.3.1. (https://github.com/visionmedia/debug/issues/797)
npm WARN deprecated request@2.88.2: request has been deprecated, see https://github.com/request/request/issues/3142
npm WARN deprecated core-js@2.6.12: core-js@<3 is no longer maintained and not recommended for usage due to the number of issues. Please, upgrade your dependencies to the actual version of core-js@3.
npm ERR! code 1
npm ERR! path /home/froyer/src/libp2p/js-libp2p
npm ERR! command failed
npm ERR! command sh -c aegir build --no-bundle
npm ERR! Error: Cannot find module 'es6-promisify'
npm ERR! Require stack:
npm ERR! - /home/froyer/src/libp2p/js-libp2p/src/nat-manager.js
npm ERR! - /home/froyer/src/libp2p/js-libp2p/src/index.js
npm ERR! - /home/froyer/src/libp2p/js-libp2p/.aegir.js
npm ERR! - /home/froyer/src/libp2p/js-libp2p/node_modules/aegir/src/utils.js
npm ERR! - /home/froyer/src/libp2p/js-libp2p/node_modules/aegir/src/config/user.js
npm ERR! - /home/froyer/src/libp2p/js-libp2p/node_modules/aegir/cli.js
npm ERR!     at Function.Module._resolveFilename (node:internal/modules/cjs/loader:924:15)
npm ERR!     at Function.Module._load (node:internal/modules/cjs/loader:769:27)
npm ERR!     at Module.require (node:internal/modules/cjs/loader:996:19)
npm ERR!     at require (node:internal/modules/cjs/helpers:92:18)
npm ERR!     at Object.<anonymous> (/home/froyer/src/libp2p/js-libp2p/src/nat-manager.js:6:23)
npm ERR!     at Module._compile (node:internal/modules/cjs/loader:1092:14)
npm ERR!     at Object.Module._extensions..js (node:internal/modules/cjs/loader:1121:10)
npm ERR!     at Module.load (node:internal/modules/cjs/loader:972:32)
npm ERR!     at Function.Module._load (node:internal/modules/cjs/loader:813:14)
npm ERR!     at Module.require (node:internal/modules/cjs/loader:996:19) {
npm ERR!   code: 'MODULE_NOT_FOUND',
npm ERR!   requireStack: [
npm ERR!     '/home/froyer/src/libp2p/js-libp2p/src/nat-manager.js',
npm ERR!     '/home/froyer/src/libp2p/js-libp2p/src/index.js',
npm ERR!     '/home/froyer/src/libp2p/js-libp2p/.aegir.js',
npm ERR!     '/home/froyer/src/libp2p/js-libp2p/node_modules/aegir/src/utils.js',
npm ERR!     '/home/froyer/src/libp2p/js-libp2p/node_modules/aegir/src/config/user.js',
npm ERR!     '/home/froyer/src/libp2p/js-libp2p/node_modules/aegir/cli.js'
npm ERR!   ]
npm ERR! }
npm ERR! Error: Cannot find module 'es6-promisify'
npm ERR! Require stack:
npm ERR! - /home/froyer/src/libp2p/js-libp2p/src/nat-manager.js
npm ERR! - /home/froyer/src/libp2p/js-libp2p/src/index.js
npm ERR! - /home/froyer/src/libp2p/js-libp2p/.aegir.js
npm ERR! - /home/froyer/src/libp2p/js-libp2p/node_modules/aegir/src/utils.js
npm ERR! - /home/froyer/src/libp2p/js-libp2p/node_modules/aegir/src/config/webpack.config.js
npm ERR! - /home/froyer/src/libp2p/js-libp2p/node_modules/webpack-cli/bin/utils/convert-argv.js
npm ERR! - /home/froyer/src/libp2p/js-libp2p/node_modules/webpack-cli/bin/cli.js
npm ERR!     at Function.Module._resolveFilename (node:internal/modules/cjs/loader:924:15)
npm ERR!     at Function.Module._load (node:internal/modules/cjs/loader:769:27)
npm ERR!     at Module.require (node:internal/modules/cjs/loader:996:19)
npm ERR!     at require (/home/froyer/src/libp2p/js-libp2p/node_modules/v8-compile-cache/v8-compile-cache.js:159:20)
npm ERR!     at Object.<anonymous> (/home/froyer/src/libp2p/js-libp2p/src/nat-manager.js:6:23)
npm ERR!     at Module._compile (/home/froyer/src/libp2p/js-libp2p/node_modules/v8-compile-cache/v8-compile-cache.js:192:30)
npm ERR!     at Object.Module._extensions..js (node:internal/modules/cjs/loader:1121:10)
npm ERR!     at Module.load (node:internal/modules/cjs/loader:972:32)
npm ERR!     at Function.Module._load (node:internal/modules/cjs/loader:813:14)
npm ERR!     at Module.require (node:internal/modules/cjs/loader:996:19)
npm ERR!     at require (/home/froyer/src/libp2p/js-libp2p/node_modules/v8-compile-cache/v8-compile-cache.js:159:20)
npm ERR!     at Object.<anonymous> (/home/froyer/src/libp2p/js-libp2p/src/index.js:35:20)
npm ERR!     at Module._compile (/home/froyer/src/libp2p/js-libp2p/node_modules/v8-compile-cache/v8-compile-cache.js:192:30)
npm ERR!     at Object.Module._extensions..js (node:internal/modules/cjs/loader:1121:10)
npm ERR!     at Module.load (node:internal/modules/cjs/loader:972:32)
npm ERR!     at Function.Module._load (node:internal/modules/cjs/loader:813:14)
npm ERR!     at Module.require (node:internal/modules/cjs/loader:996:19)
npm ERR!     at require (/home/froyer/src/libp2p/js-libp2p/node_modules/v8-compile-cache/v8-compile-cache.js:159:20)
npm ERR!     at Object.<anonymous> (/home/froyer/src/libp2p/js-libp2p/.aegir.js:4:16)
npm ERR!     at Module._compile (/home/froyer/src/libp2p/js-libp2p/node_modules/v8-compile-cache/v8-compile-cache.js:192:30)
npm ERR!     at Object.Module._extensions..js (node:internal/modules/cjs/loader:1121:10)
npm ERR!     at Module.load (node:internal/modules/cjs/loader:972:32)
npm ERR!     at Function.Module._load (node:internal/modules/cjs/loader:813:14)
npm ERR!     at Module.require (node:internal/modules/cjs/loader:996:19)
npm ERR!     at require (/home/froyer/src/libp2p/js-libp2p/node_modules/v8-compile-cache/v8-compile-cache.js:159:20)
npm ERR!     at Object.exports.getUserConfig (/home/froyer/src/libp2p/js-libp2p/node_modules/aegir/src/utils.js:110:12)
npm ERR!     at userConfig (/home/froyer/src/libp2p/js-libp2p/node_modules/aegir/src/config/user.js:32:28)
npm ERR!     at Object.<anonymous> (/home/froyer/src/libp2p/js-libp2p/node_modules/aegir/src/config/webpack.config.js:9:37)
npm ERR!     at Module._compile (/home/froyer/src/libp2p/js-libp2p/node_modules/v8-compile-cache/v8-compile-cache.js:192:30)
npm ERR!     at Object.Module._extensions..js (node:internal/modules/cjs/loader:1121:10) {
npm ERR!   code: 'MODULE_NOT_FOUND',
npm ERR!   requireStack: [
npm ERR!     '/home/froyer/src/libp2p/js-libp2p/src/nat-manager.js',
npm ERR!     '/home/froyer/src/libp2p/js-libp2p/src/index.js',
npm ERR!     '/home/froyer/src/libp2p/js-libp2p/.aegir.js',
npm ERR!     '/home/froyer/src/libp2p/js-libp2p/node_modules/aegir/src/utils.js',
npm ERR!     '/home/froyer/src/libp2p/js-libp2p/node_modules/aegir/src/config/webpack.config.js',
npm ERR!     '/home/froyer/src/libp2p/js-libp2p/node_modules/webpack-cli/bin/utils/convert-argv.js',
npm ERR!     '/home/froyer/src/libp2p/js-libp2p/node_modules/webpack-cli/bin/cli.js'
npm ERR!   ]
npm ERR! }

```

--- a/examples/libp2p-in-the-browser/README.md
+++ b/examples/libp2p-in-the-browser/README.md
@@ -4,9 +4,13 @@ This example leverages the [Parcel.js bundler](https://parceljs.org/) to compile
 
 ## Setup
 
-In order to run the example, first install the dependencies from same directory as this README:
+In order to run the example:
+
+- Install dependencey at the root of the js-libp2p repository (if not already done), 
+- then, install the dependencies from same directory as this README:
 
 ```
+npm install
 cd ./examples/libp2p-in-the-browser
 npm install
 ```

